content/_data/upgrades/2-516-2.adoc

```diff
@@ -0,0 +1 @@
+No notable changes requiring upgrade notes.
```

Comment:
Looks like your source branch is not up to date with the `jenkins` remote 's `master` branch.